test/unit/alignment/pairwise/fixture/global_affine_banded.hpp

```diff
@@ -55,8 +55,10 @@ static auto dna4_01 = []()
         -18,
         "A---ACCGGTTAACCGGTT",
         "ACGTAC----------GTA",
-        seqan3::alignment_coordinate{seqan3::detail::column_index_type{0u}, seqan3::detail::row_index_type{0u}},
-        seqan3::alignment_coordinate{seqan3::detail::column_index_type{16u}, seqan3::detail::row_index_type{9u}},
+        0u,
+        0u,
+        16u,
+        9u,
```

Comment:
Maybe you can reapply your regex and add

```suggestion
        /*.sequence1_begin_position = */ 0u,
        /*.sequence2_begin_position = */ 0u,
        /*.sequence1_end_position = */ 16u,
        /*.sequence2_end_position = */ 9u,
```